shuffleAnArray.py

```diff
@@ -1,5 +1,5 @@
 class Solution(object):
-
+    import random
```

Comment:
Not here, before class definition, very top.

Reply:
sure